channel/Makefile

```diff
@@ -28,3 +28,20 @@ gen-hd:
 build-hd: gen-hd $(APPNAME)
 
 install-hd: build-hd install
+
+update_truex_lib_uri:
+	sed -i '' 's/ComponentLibrary id=\"TruexAdRendererLib\" uri=\".*\"/ComponentLibrary id=\"TruexAdRendererLib\" uri=\"http:\/\/ctv.truex.com\/roku\/v${MAJOR}_${MINOR}\/${RC_DEVELOP}\/${LIBNAME}-${TARGET_VERSION}.pkg\"/' ./components/TruexAdRendererScene.xml ;\
+	sed -i '' 's/TruexAdRenderer-availability-v1.brs/TruexAdRenderer-availability-${TARGET_VERSION}.brs/' ./source/room_main.brs ;\
```

Comment:
Taking a bit of shortcut here by not checking the full line of code.  Given that this is for test deployments, this seems fine.